bitsail-connectors/connector-redis/src/main/java/com/bytedance/bitsail/connector/redis/config/RedisOptions.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.redis.config;
+
+import com.bytedance.bitsail.connector.redis.core.jedis.JedisCommandDescription;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+
+import java.io.Serializable;
+
+@Builder
+@AllArgsConstructor
+@Data
+public class RedisOptions implements Serializable {
+  private static final long serialVersionUID = 1865789744864217944L;
+
+  private String redisHost;
+  private int redisPort;
+  private String redisPassword;
+  private int timeout;
+  /**
+   * Expiring times in seconds.
+   */
+  private int ttlInSeconds;
+
+  /**
+   * Batch send by pipeline after 'batchInterval' records.
```

Comment:
Didn't quite get this. Would you clarify

Reply:
this is the same as batch size, we use a recordQueue which size is batchInterval, when we get batchInterval records, we flush it. I change this annotation to that：
![image](https://user-images.githubusercontent.com/47937942/200471839-f034b505-705c-4672-a58c-6cfdd146a818.png)